lib/livebook_web/live/apps_live.ex

```diff
@@ -0,0 +1,155 @@
+defmodule LivebookWeb.AppsLive do
+  use LivebookWeb, :live_view
+
+  import LivebookWeb.AppHelpers
+  import LivebookWeb.SessionHelpers
+
+  alias LivebookWeb.LayoutHelpers
+  alias Livebook.Sessions
+
+  on_mount LivebookWeb.SidebarHook
+
+  @impl true
+  def mount(_params, _session, socket) do
+    if connected?(socket) do
+      Livebook.Sessions.subscribe()
+    end
+
+    sessions =
+      Sessions.list_sessions()
+      |> Enum.filter(&(&1.mode == :app))
+      |> Enum.sort_by(& &1.created_at, {:desc, DateTime})
+
+    {:ok, assign(socket, sessions: sessions, page_title: "Livebook - Apps")}
+  end
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <LayoutHelpers.layout
+      current_page={~p"/apps"}
+      current_user={@current_user}
+      saved_hubs={@saved_hubs}
+    >
+      <div class="p-4 md:px-12 md:py-7 max-w-screen-lg mx-auto space-y-4">
+        <div>
+          <LayoutHelpers.title text="Apps" />
+          <p class="mt-4 mb-8 text-gray-700">
+            <%= if @sessions == [] do %>
+              No apps currently running.
+            <% else %>
+              These apps are currently running.
```

Comment:
nitpick: If app is in the "stopped" state is it "running"? Not sure about any wording changes but I thought I'd point it out anyway. Maybe we just remove this `else` clause altogether? :)

Reply:
Good call, will improve the empty message and remove the else clause.